cmd/editorconfig/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bytes"
+	"path/filepath"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/ini.v1"
+
+	ec "github.com/editorconfig/editorconfig-core-go"
+)
+
+const (
+	Version = "1.2.0"
+)
+
+func main() {
+	var (
+		configname string
+		configversion string
+		showVersionFlag bool
+	)
+	flag.StringVar(&configname, "f", ec.ConfigNameDefault, "Specify conf filename other than `.editorconfig'")
+	flag.StringVar(&configversion, "b", "", "Specify version (used by devs to test compatibility)")
+	flag.BoolVar(&showVersionFlag, "v", false, "Display version information")
+	flag.BoolVar(&showVersionFlag, "version", false, "Display version information")
+	flag.Parse()
+
+	if showVersionFlag {
+		fmt.Printf("EditorConfig Core Go, Version %s\n", Version)
+		os.Exit(0)
+	}
+
+	rest := flag.Args()
+
+	for _, file := range rest {
+		if !filepath.IsAbs(file) {
+			fmt.Fprintf(os.Stderr, "Input file must be a full path name: %s\n", file)
+			os.Exit(1)
+		}
+		def, err := ec.GetDefinitionForFilenameWithConfigname(file, configname)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+
+		var (
+			iniFile = ini.Empty()
+			buffer  = bytes.NewBuffer(nil)
+		)
+		ini.PrettyFormat = false
+		if len(rest) < 2 {
+			def.Selector = ini.DEFAULT_SECTION
```

Comment:
Not saying something is wrong with this code, but I couldn't understand the intention of this condition

Maybe it's worth documenting the behavior of this tool on README

Reply:
This behavior is defined in core-test.

- When one file is given do not print section
- When multiple files are given, use filenames for sections

https://github.com/editorconfig/editorconfig-core-test/blob/0001aa28f3816bb72cdbea09598252b50e4695c2/cli/CMakeLists.txt#L41

So I thinks all existing core executables should meet this requirement, but documenting it to README (after) seems a good idea anyway!